veusz/windows/treeeditwindow.py

```diff
@@ -1159,6 +1167,70 @@ def slotWidgetCopy(self):
             clipboard = qt.QApplication.clipboard()
             clipboard.setMimeData(mimedata)
 
+    def slotWidgetCopyAsImage(self):
+        """Copy current page to the clipboard as image files."""
+        # general image extentions and their mime types exportable from document
+        exts = {'svg': ['image/svg+xml'], 
+                'png': ['image/png', 'PNG'], 
+                'bmp': ['image/x-bmp'],
+                }
+
+        def checkDone(export, exts):
+            """Check whether exporting pages as images has finished."""
+            if not export.haveDone():
+                return
+            try:
+                export.finish()
+                clipboard = qt.QApplication.clipboard()
+                data = qt.QMimeData()
+
+                # set images to QMimeData
+                for ext, mimes in exts.items():
+                    try:
+                        fname = tmpdir.name + '/tmp.{}'.format(ext)
```

Comment:
Change to os.path.join(tmpdir.name, 'tmp.{}'.format(ext))?

Reply:
Thanks for pointing out the buggy code.
I fixed.